### PR TITLE
Switch risk level over to annotation

### PR DESCRIPTION
**Pull request by jasonaowen**
Pluralize the table name, remove the `lu_` prefix, and add seed data.

Issue #36 Use Hibernate 5, instead of 4

### CODE REVIEW

File: psm-app/services/src/main/java/gov/medicaid/entities/RiskLevel.java [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /*
  * Copyright 2012-2013 TopCoder, Inc.
  *
- * This code was developed under U.S. government contract NNH10CD71C.
```
Comment (kfogel) [inline psm-app/services/src/main/java/gov/medicaid/entities/RiskLevel.java]: Spurious whitespace change here.

**Comment by kfogel** [comment]
If this is just part of getting the Hibernate migration done, then +1.  If it represents our first attempt at re-designing how the PSM tracks risk levels, then we should discuss more :-).

Where I think we want to go is to have RiskLevel be a class, maybe?  That way, we can later represent different kinds of risk as different attributes/fields on that class.  For now, we can have a method that just resolves the entire class to a single word: "High", "Medium", or "Low" (or to a code that is then translated to the right localized word -- you know what I mean).  But I don't think we want to bake the 3-level thing into the code in any permanent way.

Regarding the code change: I asked in IRC what "annotation" meant in this context, and you answered helpfully:

> https://en.wikipedia.org/wiki/Java_annotation , specifically JPA annotations ( https://en.wikipedia.org/wiki/Java_Persistence_API )
> compare https://github.com/OpenTechStrategies/psm/pull/64 or https://github.com/OpenTechStrategies/psm/pull/66

...and then you said:

> some of these are pretty mechanical; just necessary steps to get to the more complicated stuff

...which makes me think this change really is just "let's get past the Hibernate migration".  In which case, sure, but let's remember this is tech debt w.r.t. how we actually want to handle risk levels.

**Comment by jasonaowen** [comment]
> If this is just part of getting the Hibernate migration done, then +1. If it represents our first attempt at re-designing how the PSM tracks risk levels, then we should discuss more :-).

It is definitely part of the Hibernate migration and not a redesign. There is no behavior changed associated with this; it is just one of the entities that needs to be converted to Hibernate 5, and it enables other classes which include it to also be converted to Hibernate 5.

> Where I think we want to go is to have RiskLevel be a class, maybe? That way, we can later represent different kinds of risk as different attributes/fields on that class. For now, we can have a method that just resolves the entire class to a single word: "High", "Medium", or "Low" (or to a code that is then translated to the right localized word -- you know what I mean). But I don't think we want to bake the 3-level thing into the code in any permanent way.

I'd say you've just described how it works right now. The 3-level thing is data in the database, and it is an attribute of `ProviderProfile` (which is why I converted this class, so that I can convert `ProviderProfile`). I think we could later introduce various attributes of provider profiles that each have their own `RiskLevel`, and doing so probably wouldn't require any changes to `RiskLevel`.

File: psm-app/services/src/main/java/gov/medicaid/entities/RiskLevel.java (hunk 1 above)
Comment (jasonaowen) [inline psm-app/services/src/main/java/gov/medicaid/entities/RiskLevel.java]: Fixed.

**Comment by jasonaowen** [comment]
Updated to remove spurious whitespace change.

**Pull request merged**